dataset: "FusionPortableV2"
common:
    PointCloudTopic: "/os_cloud_node/points"
    ImageTopic: "/img_depth"
    RawImageTopic: "/raw_img"
# We recommend using it when your sensor noise is severe.
high_noise: false
mean: 4.8
# inc of pixel and point
inc:
    pixel_inc: 1.5
    range_inc: 0.02
    degree_inc: 0.05
width: 1024
height: 768
# Initial extrinsic 
Intrinsic: [602.4532, 0.000000e+00, 520.8442,
            0.000000e+00, 602.4183, 397.5420,
            0.000000e+00, 0.000000e+00, 1.000000e+00]
DistCoeff: [-0.0891, 0.0885, 0.0005, 0.0020, 0.0]
Extrinsic: [-0.0072456, -0.9997507, -0.0211169, 0.0818,
            0.0051766, 0.0210797,-0.9997644, -0.0522,
            0.9999604, -0.0073532, 0.0050226, -0.0394]


